site/content/en/docs/concepts/cohort.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Cohort"
+date: 2025-04-16
+weight: 4
+description: >
+  A cluster-scoped resource for organizing quotas
+---
+
+## Hello, Cohorts
+Cohorts give you the ability to organize your Quotas. ClusterQueues within the same Cohort (or same CohortTree for [Hieararchical Cohorts](#hierarchical-cohorts)) can share resources with each other. The simplest possible Cohort is the following:
```

Comment:
It would be better to clarify what `same CohortTree` is, or just use `siblings from same parent Cohort` instead of the noun `CohortTree`, IMO